### DISCUSSION TITLE
Saleor-Dashboard Logo

### DISCUSSION THREAD

**Discussion by a-3140** [saleor/saleor#10424] [General]
1. Are we allowed to change the logo on the dashboard legally by forking the macaw-ui?

2. Can we contribute to the macaw-ui to have the logo be an optional prop on the sidebar?

Note: Not sure if I posted on the right one please guide me to the correct discussion/chat area if needed be. Thanks!

Found a similar discussion but tagged under deployment.

https://github.com/saleor/saleor/discussions/10231

**Comment by peelar** [comment]
Hello @a-3140 👋 

1. Yes, you can fork `macaw-ui` to change the logo. In fact, it is the only way to do so right now.
2. You are welcome to contribute to any of our repositories. Each PR, however, has to align with the Saleor vision and backlog, so I am not able to guarantee if such a feature would get merged. You can also submit a "feature request" issue and we will consider it.

Please let us know if you have any other questions. Have a nice day 🌻

**Reply by looped-monk** [reply]
Hi @peelar @a-3140  I have placed my logos in /storybook/public and replaced the default logos (favicon, msite etc) with exactly the same dimensions and uploaded to my github npm packages for version 1.4.2. Only the favicon changes when i open the saleor dashboard. The default saleor logo doesn't get replaced. Anything that I'm missing here.
